argus/recipes/cloud/windows.py

```diff
@@ -43,7 +43,9 @@ def wait_for_boot_completion(self):
 
     def execution_prologue(self):
         LOG.info("Retrieve common module for proper script execution.")
-
+        set_mtu_cmd = ('netsh interface ipv4 set subinterface "Ethernet" '
```

Comment:
Put this code block in a different method and catch all the exceptions that may occur.